hw/ip/kmac/lint/sha3.waiver

```diff
@@ -15,3 +15,12 @@ waive -rules {RESET_USE} -location {keccak_round.sv} -regexp {'rst_ni' is connec
 waive -rules {TWO_STATE_TYPE} -location {keccak_2share.sv} \
       -regexp {'index_z' is of two state} \
       -comment "index_z behaves as constant"
+
+waive -rules {ARITH_CONTEXT} -location {sha3pad.sv} -regexp {.*Bitlength of arithmetic operation.*is self-determined in this context.*} \
+      -comment "Carry issue when determning the bit length."
+
+waive -rules {PARAM_NOT_USED} -location {keccak_round.sv} -regexp {.*Parameter.*not used in module.*} \
+      -comment "Parameter not used in this instantiation."
```

Comment:
This would completely waive the warning. Can you specialize it to the parameter in question?

Reply:
Narrowed down the waiver to just this specific parameter name.